test/enqueue_links.js

```diff
@@ -6,6 +6,30 @@ import { RequestQueue } from '../build/request_queue';
 
 const { utils: { log } } = Apify;
 
+const HTML = `
+<html>
+    <head>
+        <title>Example</title>
+    </head>
+    <body>
+        <p>
+            The ships hung in the sky, much the <a class="click" href="https://example.com/a/b/first">way that</a> bricks don't.
+        </p>
+        <ul>
+            <li>These aren't the Droids you're looking for</li>
+            <li><a href="https://example.com/a/second">I'm sorry, Dave. I'm afraid I can't do that.</a></li>
+            <li><a class="click" href="https://example.com/a/b/third">I'm sorry, Dave. I'm afraid I can't do that.</a></li>
+        </ul>
+        <a class="click" href="https://another.com/a/fifth">The Greatest Science Fiction Quotes Of All Time</a>
+        <p>
+            Don't know, I don't know such stuff. I just do eyes, ju-, ju-, just eyes... just genetic design,
+            just eyes. You Nexus, huh? I design your <a class="click" href="http://cool.com/">eyes</a>.
+        </p>
+        <a href="/x/relative">This is a relative link.</a>
```

Comment:
This is a test for **absolute path**. Can we also add a test for relative path, i.e. `x/relative` (without leading `/`)

Reply:
I might be wrong but I need to try it in the browser.

Reply:
Check relative path section here - https://www.webucator.com/tutorial/learn-html/html-links.cfm